src/main/resources/lib/credentials/select.jelly

```diff
@@ -107,11 +107,11 @@
           <input type="hidden" name="includeUser" value="${includeUser}"/>
         </j:otherwise>
       </j:choose>
-      <div class="form-group jenkins-form-item jenkins-form-item--tight">
-        <f:select clazz="${attrs.clazz} credentials-select" field="${attrs.field}"
```

Comment:
**Hide whitespace** to review

Reply:
> **Hide whitespace** to review

Agreed.  Much clearer with [white space diffs suppressed](https://github.com/jenkinsci/credentials-plugin/pull/316/files?w=1).